app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,27 @@
 module Api
   class AuthenticationsController < BaseController
+    def create_resource(_type, _id, data)
+      attrs = validate_auth_attrs(data)
+      klass = ::Authentication.class_from_request_data(attrs)
+      # TODO: Temporary validation - remove
+      raise 'type not currently supported' unless klass.respond_to?(:create_in_provider_queue)
+      task_id = klass.create_in_provider_queue(attrs['manager_resource'], attrs.except('type', 'manager_resource'))
+      MiqTask.find(task_id)
```

Comment:
This and the equivalent return as subcollection create should return the action_result signature which supports tasks, i.e action_result(true, desc, :task_id => task_id).   create return signatures is either object created or action result, Thanks.